include/tvm/auto_scheduler/measure.h

```diff
@@ -340,9 +351,14 @@ class LocalBuilderNode : public ProgramBuilderNode {
  public:
   /*! \brief Build function. */
   String build_func;
+  /*! \brief Functor with python implementation of submit method. */
+  PackedFunc submit_func;
```

Comment:
Could this work well on the remote or this only happens on the host side?

Reply:
No, it will not. This function is only for host side. Will never transferred to execute on remote device.